pipelines/skylab/scATAC/scATAC.wdl

```diff
@@ -291,7 +291,7 @@ task MakeCompliantBAM {
 task BreakoutSnap {
     input {
         File snap_input
-        String docker_image = "us.gcr.io/broad-gotc-prod/pytools:1.0.0-1661263730"
+        String docker_image = "us.gcr.io/broad-gotc-prod/warp-tools:1.0.1-1686932671"
```

Comment:
Can you update the link to the python script for this task here?: https://github.com/broadinstitute/warp/blob/develop/website/docs/Pipelines/Single_Cell_ATAC_Seq_Pipeline/README.md#makecompliantbam

Reply:
Sure thing! Thanks Kaylee!

Reply:
Hi Kaylee. I made the changes. Thanks!